packages/govuk-frontend/src/govuk/tools/rebrand.unit.test.js

```diff
@@ -0,0 +1,145 @@
+const { compileSassString } = require('@govuk-frontend/helpers/tests')
+const { outdent } = require('outdent')
+
+describe('@mixin _govuk-rebrand', () => {
+  it('wraps arbitrary properties in a class', async () => {
+    const sass = `
+      $govuk-suppressed-warnings: ("legacy-organisation-colours");
```

Comment:
How necessary is it to suppress the warning on each of the tests? Is it interfering with the tests for error throwing?

Reply:
It was more for convenience when the tests fail, as the console gets really crowded because of that warning and it makes it a little tricky to separate what's the Sass warning and what's the test error.

Reply:
Fair enough! 👍